kore/src/Kore/Attribute/Symbol.hs

```diff
@@ -108,29 +108,29 @@ view the effective attributes, use the functions defined in this module.
  -}
 data Symbol =
     Symbol
-    { function      :: !Function
+    { function       :: !Function
       -- ^ Whether a symbol represents a function
-    , functional    :: !Functional
+    , functional     :: !Functional
       -- ^ Whether a symbol is functional
-    , constructor   :: !Constructor
+    , constructor    :: !Constructor
       -- ^ Whether a symbol represents a constructor
-    , injective     :: !Injective
+    , injective      :: !Injective
       -- ^ Whether a symbol represents an injective function
-    , sortInjection :: !SortInjection
+    , sortInjection  :: !SortInjection
       -- ^ Whether a symbol is a sort injection
-    , anywhere      :: !Anywhere
-    , hook          :: !Hook
+    , anywhere       :: !Anywhere
+    , hook           :: !Hook
       -- ^ The builtin sort or symbol hooked to a sort or symbol
-    , smtlib        :: !Smtlib
-    , smthook       :: !Smthook
-    , memo          :: !Memo
-    , klabel        :: !Klabel
-    , symbolKywd    :: !SymbolKywd
-    , noEvaluators  :: !NoEvaluators
-    , unitHook      :: !(Unit SymbolOrAlias)
-    , elementHook   :: !(Element SymbolOrAlias)
-    , concatHook    :: !(Concat SymbolOrAlias)
-    -- ^ The above three are only populated if the symbol is a concat
+    , smtlib         :: !Smtlib
+    , smthook        :: !Smthook
+    , memo           :: !Memo
+    , klabel         :: !Klabel
+    , symbolKywd     :: !SymbolKywd
+    , noEvaluators   :: !NoEvaluators
+    , builtinUnit    :: !(Unit SymbolOrAlias)
+    , builtinElement :: !(Element SymbolOrAlias)
+    , builtinConcat  :: !(Concat SymbolOrAlias)
+    -- ^ The above three are only populated if the symbol is a builtinAc symbol
```

Comment:
I don't understand what is "built-in" about these symbols. They are user-defined symbols only used for unparsing. (Now that I think of it, the comment should mention these are only used for unparsing.) If simply `unit`, `element`, and `concat` cause too many collisions, the next best names would be `unitSymbol`, etc. As for the documentation, what is a "builtinAc symbol"?

Reply:
The fields are "built-in" in the sense that they represent the built-in symbol labels but I don't mind changing them if we find a better name for them. `unitSymbol`  and the rest are also unavailable due to import conflicts from `Unit`, `Element` and `Concat`.
And a built-in Ac symbol is a symbol of a built-in associative-commutative type. I tried to use the same naming convention that we use for `InternalAc`, NormalizedAc` etc.